### ISSUE TITLE
Color contrast problems for people with colorblindness issues

### ISSUE THREAD

**Issue by HiEv** [klembot/twinejs#611]
The colors used for syntax highlighting do not have sufficient contrast for some people with colorblindness issues to be able to read some parts.

For example, [aliengeo posted to the IntFiction.org site](https://intfiction.org/t/custom-syntax-highlighting-for-twine-2-stylesheet/42716) this image, representing how element IDs look to him in the Stylesheet section:
![Impaired_view](https://user-images.githubusercontent.com/38147576/66027590-3b8a7580-e4c9-11e9-8ba9-1ad73982cd7d.png)

The issue is that the blue (#3300AA) text used for element IDs on that dark-gray (#262626) background color has a color contrast ratio of 1.24:1, when 3:1 is the bare minimum recommended for large text, and 4.5:1 is the bare minimum for normal text.  A 7:1 contrast ratio or higher is ideal.

See the [WebAIM Contrast Checker](https://webaim.org/resources/contrastchecker/) for details.

Here are some (but not necessarily all) of the colors which have problems (i.e. a contrast ratio of less than 4.5:1):
```
.cm-s-default .cm-builtin {
    color: #30a;
}
.cm-keyword {
	color: #c66;
}
.cm-comment {
	color: #a3685a;
}
```
And here's what they would be when brightened to a contrast ratio of ~4.5:1:
```
.cm-s-default .cm-builtin {
    color: #9F75FF;
}
.cm-keyword {
	color: #D07171;
}
.cm-comment {
	color: #B58278;
}
```

**Comment by klembot** [comment]
Curious if there is a CodeMirror theme that is known to be accessible that we can use.

**Comment by HiEv** [comment]
This should fix everything (changes will have uppercase hexcode colors):
```
.cm-s-default {
	.CodeMirror {color: #282a2e;}
 	div.CodeMirror-selected {background: #d6d6d6 !important;}
 	.CodeMirror-gutters {background: #ffffff; border-right: 0px;}
 	.CodeMirror-linenumber {color: #6F766F;}
 	.CodeMirror-cursor {border-left: 1px solid #969896 !important;}

	span.cm-comment {color: #A16759;}
	span.cm-atom {color: #91639C;}
	span.cm-number {color: #91639C;}

	span.cm-property, span.cm-attribute {color: #717935;}
	span.cm-keyword {color: #C44F4F;}
	span.cm-string {color: #9C6E11;}

	span.cm-variable {color: #717935;}
	span.cm-variable-2 {color: #507A9B;}
	span.cm-def {color: #B25F24;}
	span.cm-error {border-bottom: 1px dotted #cc6666;}
	span.cm-bracket {color: #282a2e;}
	span.cm-tag {color: #C44F4F;}
	span.cm-link {color: #91639C;}

	.CodeMirror-matchingbracket {text-decoration: underline; color: white !important;}
}

body.theme-dark .cm-s-default {
	.CodeMirror {color: #e0e0e0;}
	div.CodeMirror-selected {background: #4d4d4c !important;}
	.CodeMirror-gutters {background: #1d1f21; border-right: 0px;}
	.CodeMirror-linenumber {color: #969896;}
	.CodeMirror-cursor {border-left: 1px solid #b4b7b4 !important;}

	span.cm-comment {color: #B58278;}
	span.cm-atom {color: #b294bb;}
	span.cm-number {color: #b294bb;}

	span.cm-property, span.cm-attribute {color: #b5bd68;}
	span.cm-keyword {color: #D07171;}
	span.cm-string {color: #f0c674;}

	span.cm-variable {color: #b5bd68;}
	span.cm-variable-2 {color: #81a2be;}
	span.cm-def {color: #de935f;}
	span.cm-error {border-bottom: 1px dotted #cc6666;}
	span.cm-bracket {color: #e0e0e0;}
	span.cm-tag {color: #D07171;}
	span.cm-link {color: #b294bb;}

	.CodeMirror-matchingbracket {text-decoration: underline; color: white !important;}
}
```
I'm not sure why both `.CodeMirror-matchingbracket` lines use `color: white !important;`, when I'd expect one of them to use black, but it doesn't appear to affect anything, so it can probably be ignored.

**Comment by klembot** [comment]
Would you be comfortable doing a PR for this?

**Comment by HiEv** [comment]
Sure.  It looks like it's just twinejs/src/vue/[codemirror-theme.less](https://github.com/klembot/twinejs/blob/master/src/vue/codemirror-theme.less) that needs to be modified, along with some of the CodeMirror/lib/[codemirror.css](CodeMirror/lib/codemirror.css) defaults added/overridden.

**Comment by klembot** [comment]
Syntax colors were updated in 2.4--if they have contrast issues, please open an issue related to that.
